src/Components/Patient/PatientRegister.tsx

```diff
@@ -537,7 +549,9 @@ export const PatientRegister = (props: PatientRegisterProps) => {
             state.form.nationality === "India" &&
             !Number(state.form[field])
           ) {
-            errors[field] = "Please select ward";
+            errors[field] = (
+              <span className="text-red-500">Please select ward</span>
```

Comment:
@siva-balan wouldn't the FormField component apply the colors automatically?

You can make use of `SelectFormField` for gender select.

```suggestion
              <span className="text-red-500">Please select ward</span>
```

```suggestion
              Please select ward
```